content/en/developers/marketplace.md

```diff
@@ -108,36 +108,43 @@ Technology Partners can request access to the private Marketplace repository by
 
 The Datadog Development Toolkit command (`ddev`) allows you to create scaffolding when you are first developing your integration, by spinning up a skeleton of all the assets and metadata for your tile.
 
-Ensure that Python 3.8 or higher is installed.	
+Ensure that Python 3.8 or higher is installed (https://www.python.org/downloads/).	
 
-The latest released version may be installed from [PyPI][23]:
+To avoid a range of potential environment conflicts, we recommend going into the directory where you cloned the repository and creating a simple virtual environment with the following command: `python3 -m pip install virtualenv --user`
 
-`python -m pip install --upgrade "datadog-checks-dev[cli]`
+The latest released version of the Datadog Development Toolkit may then be installed from [PyPI][23]:
+
+`python -m pip install --upgrade "datadog-checks-dev[cli]"`
+
+**Note:** if you are using Z shell, you may need to use escape characters: `python -m pip install --upgrade datadog-checks-dev\[cli\]` 
 
 Set the location of to the cloned repository:
 
 #### Marketplace:
 
-`ddev config set marketplace /path/to/marketplace`
+`ddev config set marketplace /path/to/marketplace_directory` (For example: /Users/yourname/Documents/marketplace)
 
 `ddev config set repo marketplace`
 
 #### Integrations-Extras:
 
-`ddev config set extras /path/to/integrations-extras`
+`ddev config set extras /path/to/integrations-extras_directory` (For example: /Users/yourname/Documents/integrations-extras)
 
 `ddev config set repo extras`
 
 #### Tile Only Listing
 
 For standalone software and services--or if your integration is using the Datadog API and does not contain any Python code--the Development Toolkit supports a tile-only command. 
-`ddev create -t` 
+
+`cd` into the marketplace or integrations-extras directory and run: 
+
+`ddev create -t tile "<Offering Name>"`
```

Comment:
Use three-backticks Markdown code-fence for the full-line monospace, not singles

Reply:
Added the three backticks throughout the doc. Hopefully, I didn't miss any :)